modules/nanobind_bazel/2.4.0/presubmit.yml

```diff
@@ -0,0 +1,18 @@
+matrix:
+  bazel:
+    - 7.x
+    - 8.x
+  platform:
+    - debian10
+    - ubuntu2004
+    - macos
+    - macos_arm64
+    - windows
+
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+    - '@nanobind_bazel//:nanobind'
```

Comment:
```suggestion
    build_targets:
    - '@nanobind_bazel//:nanobind'
    build_flags:
    - --cxxopt=-std=c++17
```

Reply:
Are those cross-platform? Then I would also have to put `--cxxopt=/std:c++17` for MSVC/Windows most likely.

Reply:
Resolved by unrolling the platform and jobs.